pkg/standalone/standalone.go

```diff
@@ -223,6 +249,7 @@ func Init(runtimeVersion, dashboardVersion string, dockerNetwork string, slimMod
 	}
 
 	info := initInfo{
+		bundleDet:        &bundleDet, // will be nil if fromDir is empty, so must be used in conjunction with fromDir.
```

Comment:
```suggestion
		bundleDet:        &bundleDet, // must be used in conjunction with fromDir.
```

&bundleNet won't be nil even if --from-dir flag is not there

Reply:
addressed comment.